### DISCUSSION TITLE
Unable to download 0.3.0 version installer

### DISCUSSION THREAD

**Discussion by kasper77pl** [ExistentialAudio/BlackHole#522] [General]
Hello Devin

Thanks for your great work !

I'd like to let you know that on this link: 
https://existential.audio/blackhole/?pk_campaign=github&pk_kwd=readme

I can download older version 0.2.10 only:
Screen below:

![image](https://user-images.githubusercontent.com/72818087/149340376-cb1b5e77-6595-41d3-b01a-734477529a2f.png)

**Comment by devinroth** [comment]
Yes. I haven’t updated the installer yet.

Devin Roth
